### DISCUSSION TITLE
An idea to declutter the pages

### DISCUSSION THREAD

**Discussion by stonegrift** [JVital2013/vitality-goes#77] [Ideas]
Again I'm not up on PHP.  A situation came up with an unexpected folder called "Unknown" being generated by Satdump.  Looking at the image titles and comparing them the data I found here, https://docs.opendata.aws/noaa-goes16/cics-readme.html  I started building a new ini file for all the Level 2 images since that seemed to be what I was receiving.  Turns out there are 58 different types of  level 2 images and that would really cramp up the new ini page of images.  So I thought PHP could be used to check if there are any images. If it is null, then it won't show the image type from the ini.  The down side is potentially you could get a new user with a bunch of blank pages.

**Comment by JVital2013** [comment]
I don't like the idea of hiding cards if no images are shown. In my opinion, the point of configuring images in the ini is to say you want to display them. If you don't want an image to be shown - don't have it in the ini. If I have an image configured but it's not available, I want to know it.

That being said, two additional tidbits:
- The unknown folder in SatDump is currently unsupported because it's unsupported in SatDump (and other issues that arise around that). Once support is added in SatDump (or we determine it won't be added, which is unlikely), I'll properly support the unknown images in Vitality GOES. Until then - any configs you make won't work quite right.
- While there are many Non-CMIP image types, there are only 7 over HRIT - ACHA, ACHT, DSI, RRQPE, SST, LST, and TPW. There used to be an additional one, but it was recently discontinued on HRIT
